### ISSUE TITLE
Errors in logs

### ISSUE THREAD

**Issue by xkrt** [floragunncom/search-guard#662]
Hi! I have strange errors in logs:
```
java.lang.Exception                                                                                                                                                                                                              [45/1770]
        at com.floragunn.searchguard.auth.BackendRegistry.authz(BackendRegistry.java:659)
        at com.floragunn.searchguard.auth.BackendRegistry.access$000(BackendRegistry.java:73)
        at com.floragunn.searchguard.auth.BackendRegistry$8.call(BackendRegistry.java:701)
        at com.floragunn.searchguard.auth.BackendRegistry$8.call(BackendRegistry.java:694)
        at com.google.common.cache.LocalCache$LocalManualCache$1.load(LocalCache.java:4875)
        at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3527)
        at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2276)
...
...
```

Looks like this errors arise because of this weird code: https://github.com/floragunncom/search-guard/blob/7e0be6da4324f34f5c770319b605e0fc04ce0081/src/main/java/com/floragunn/searchguard/auth/BackendRegistry.java#L659

What this code for? It is strange that it create the empty exception object and just print strack trace of it.

Search Guard version: 24.2
Elasticsearch version: 6.6.2
Number of nodes in your cluster: 3

**Comment by floragunncom** [comment]
This is a known issue. The statement will be removed in the next release. You can ignore the exception.

**Comment by floragunncom** [comment]
Fixed with 24.3
